Author: Michael Fiedler
Date:   Mon Feb 22 19:21:20 2016 +0100

    Adapt shortcuts to old behaviour

diff --git a/shell/ev-application.c b/shell/ev-application.c
index d0f1a52..d9a5a0f 100644
--- a/shell/ev-application.c
+++ b/shell/ev-application.c
@@ -1051,6 +1051,7 @@ ev_application_startup (GApplication *gapplication)
           "win.open-copy",              "<Ctrl>N", NULL,
           "win.save-copy",              "<Ctrl>S", NULL,
           "win.print",                  "<Ctrl>P", NULL,
+          "win.show-properties",        "<alt>Return", NULL,
           "win.copy",                   "<Ctrl>C", "<Ctrl>Insert", NULL,
           "win.select-all",             "<Ctrl>A", NULL,
           "win.save-settings",          "<Ctrl>T", NULL,
@@ -1071,11 +1072,13 @@ ev_application_startup (GApplication *gapplication)
           "win.sizing-mode::fit-width", "w", NULL,
           "win.open-menu",              "F10", NULL,
           "win.caret-navigation",       "F7", NULL,
-          "win.zoom-in",                "plus", "<Ctrl>plus", "KP_Add", "<Ctrl>KP_Add", NULL,
+          "win.zoom-in",                "plus", "<Ctrl>plus", "KP_Add", "<Ctrl>KP_Add", "equal", "<Ctrl>equal", NULL,
           "win.zoom-out",               "minus", "<Ctrl>minus", "KP_Subtract", "<Ctrl>KP_Subtract", NULL,
           "win.show-side-pane",         "F9", NULL,
           "win.fullscreen",             "F11", NULL,
           "win.presentation",           "F5", NULL,
+          "win.continuous",             "c", NULL,
+          "win.dual-page",              "d", NULL,
           "win.rotate-left",            "<Ctrl>Left", NULL,
           "win.rotate-right",           "<Ctrl>Right", NULL,
           "win.inverted-colors",        "<Ctrl>I", NULL,
